docs/configuration.md

```diff
@@ -40,67 +40,72 @@ Mandatory parameters are marked as **Required**, which means that they are requi
-| `order_types` | Configure order-types depending on the action (`"buy"`, `"sell"`, `"stoploss"`, `"stoploss_on_exchange"`). [More information below](#understand-order_types). [Strategy Override](#parameters-in-the-strategy).
-| `order_time_in_force` | Configure time in force for buy and sell orders. [More information below](#understand-order_time_in_force). [Strategy Override](#parameters-in-the-strategy).
-| `exchange.name` | **Required.** Name of the exchange class to use. [List below](#user-content-what-values-for-exchangename).
-| `exchange.sandbox` | Use the 'sandbox' version of the exchange, where the exchange provides a sandbox for risk-free integration. See [here](sandbox-testing.md) in more details.
-| `exchange.key` | API key to use for the exchange. Only required when you are in production mode. **Keep it in secrete, do not disclose publicly.**
-| `exchange.secret` | API secret to use for the exchange. Only required when you are in production mode. **Keep it in secrete, do not disclose publicly.**
-| `exchange.password` | API password to use for the exchange. Only required when you are in production mode and for exchanges that use password for API requests. **Keep it in secrete, do not disclose publicly.**
-| `exchange.pair_whitelist` | List of pairs to use by the bot for trading and to check for potential trades during backtesting. Not used by VolumePairList (see [below](#dynamic-pairlists)).
-| `exchange.pair_blacklist` | List of pairs the bot must absolutely avoid for trading and backtesting (see [below](#dynamic-pairlists)).
-| `exchange.ccxt_config` | Additional CCXT parameters passed to the regular ccxt instance. Parameters may differ from exchange to exchange and are documented in the [ccxt documentation](https://ccxt.readthedocs.io/en/latest/manual.html#instantiation)
-| `exchange.ccxt_async_config` | Additional CCXT parameters passed to the async ccxt instance. Parameters may differ from exchange to exchange  and are documented in the [ccxt documentation](https://ccxt.readthedocs.io/en/latest/manual.html#instantiation)
-| `exchange.markets_refresh_interval` | The interval in minutes in which markets are reloaded. <br>*Defaults to `60` minutes.*
+| `max_open_trades` | **Required.** Number of trades open your bot will have. If -1 then it is ignored (i.e. potentially unlimited open trades).<br> ***Datatype:*** *Positive integer (-1 to use `"unlimited"` trades).*
+| `stake_currency` | **Required.** Crypto-currency used for trading. [Strategy Override](#parameters-in-the-strategy). <br> ***Datatype:*** *String*
+| `stake_amount` | **Required.** Amount of crypto-currency your bot will use for each trade. Set it to `"unlimited"` to allow the bot to use all available balance. [More information below](#understand-stake_amount). [Strategy Override](#parameters-in-the-strategy). <br> ***Datatype:*** *Positive number or `"unlimited"`.*
```

Comment:
"Positive integer or float" ?

Reply:
stake_amount is not an integer - (Think about BTC - where 1 Integer is several 1000$ ... )
float / number i'm not sure - float won't mean much to non-programmers ... (neither will integer ...)

Reply:
Can also be integer, for USDT -- like 100 USDT... It's a number in json IIRC, so the schema allows integers as well. Btw, if this is allowed now, an interger is set and not casted to a float, this could result in errors (like multiplying to integer number of trades and then divided by an integer somewhere in the code...)

Should be casted to float somewhere in configuration.py if this is not done, imho... Not in this PR, should be checked in the code. But integer is accepted and is to be accepted in the config and in the strategy)

Reply:
Or -1 can be marked as Deprecated, to have users time to adapt to new restrictions for this value. Not in this PR in this case.

Reply:
yes it is ... but AFAIK, an integer is treated as float if used in calculations and one of the 2 parties is an integer ... so i don't see this create a problem.

At least that's how it is in python3 (i think this was different in python2 - but i don't really care since we're not supporting that anyway) - i just tested ...